internal/core/logger_test.go

```diff
@@ -0,0 +1,49 @@
+package core
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/scaleway/scaleway-cli/internal/args"
+	"github.com/scaleway/scaleway-sdk-go/logger"
+)
+
+func testLogCommands() *Commands {
+	return NewCommands(
+		&Command{
+			Namespace:            "log",
+			ArgsType:             reflect.TypeOf(args.RawArgs{}),
+			AllowAnonymousClient: true,
+			Run: func(ctx context.Context, argsI interface{}) (i interface{}, e error) {
+				return "Hello World!", nil
+			},
+		},
+	)
+}
+
+func Test_Logger(t *testing.T) {
+	t.Run("Simple", Test(&TestConfig{
+		Commands: testLogCommands(),
+		BeforeFunc: func(ctx *BeforeFuncCtx) error {
+			ctx.Logger.level = logger.LogLevelDebug
```

Comment:
Log are not include in golden file so this do not test anything

Reply:
Could we change the `ctx.Logger.writer` to an `io.Writer` that writes on a file. Then we could check with an assert function that this file matches what the logger should have written? Like a golden log file?